File: Tutorial_Sample_Code/ReducedCode/ReportData/VB/VB_Win_ReducedCode_ReportData/Form1.vb
Public Class Form1

    Private Sub myCrystalReportViewer_Drill(ByVal source As System.Object, ByVal e As CrystalDecisions.Windows.Forms.DrillEventArgs) Handles myCrystalReportViewer.Drill
        drillLabel.Text = e.NewGroupName
    End Sub
End Class

File: Tutorial_Sample_Code/ReducedCode/ReportData/CS/CS_Win_ReducedCode_ReportData/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace CS_Win_ReducedCode_ReportData
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void crystalReportViewer_Drill(object source, CrystalDecisions.Windows.Forms.DrillEventArgs e)
        {
            drillLabel.Text = "You drilled down on: " + e.NewGroupName;
        }
    }
}